### DISCUSSION TITLE
连接被重置

### DISCUSSION THREAD

**Discussion by vvanglro** [go-gost/gost#487] [Q&A]
```sh
gost -L=:1080
```
版本2和3都试过，使用http代理的时候，浏览器访问Google和YouTube都会显示ERR_CONNECTION_RESET，换成socks5就正常了

这是为什么啊大佬🙇

**Comment by vvanglro** [comment]
貌似知道问题了，在使用http代理的时候，用的内部ip和客户端建立的连接，所以导致连接重置。
这是为啥啊。。
![image](https://github.com/go-gost/gost/assets/43594924/3b806c2b-c2c5-4bf0-94b3-49b85256e7ab)

**Comment by ginuerzh** [comment]
如果服务器在国外，不要使用明文协议。

**Reply by vvanglro** [reply]
香港的服务器，换成ss了。
但是那个问题没想通，为啥用http代理会导致ERR_CONNECTION_RESET
